### ISSUE TITLE
apt-get install pup

### ISSUE THREAD

**Issue by fracz** [ericchiang/pup#129]
Do you consider adding an apt repository? ;-)

**Comment by SuperSandro2000** [comment]
You mean a ppa or something like reprepro?

**Comment by fracz** [comment]
Yes, something more convenient than downloading and putting it into `/usr/bin` manually.

**Comment by SuperSandro2000** [comment]
You should put it in ``/usr/local/bin``.
BTW you can redirect curl to put a binary directly into place.

**Comment by fracz** [comment]
I know, but it's more difficult for not-programmer-end-user :-(

**Comment by merwok** [comment]
pup is now in Debian testing!

**Comment by janwerkhoven** [comment]
Currently on Debian GNU/Linux 10 (buster):
```
sudo apt-get update
sudo apt-get install pup                                                                                                                                                                    
E: Unable to locate package pup
```
Keen to use `pup` on Debian and Ubuntu servers!

Does anyone know how to install `pup` in the meanwhile without `apt-get`? I'm not getting far with the `go` command... (noob).

**Comment by merwok** [comment]
It’s in testing (bullseye)!

**Comment by jxu** [comment]
It's not in ubuntu packages https://launchpad.net/ubuntu/+source/pup

**Comment by gsamansharma** [comment]
sudo wget https://github.com/ericchiang/pup/releases/download/v0.4.0/pup_v0.4.0_linux_amd64.zip |sudo unzip pup_v0.4.0_linux_amd64.zip -d /usr/local/bin

**Comment by gsamansharma** [comment]
Paste this command in your terminal to install it 
`sudo wget https://github.com/ericchiang/pup/releases/download/v0.4.0/pup_v0.4.0_linux_amd64.zip | sudo unzip pup_v0.4.0_linux_amd64.zip -d /usr/local/bin
`

**Comment by meleu** [comment]
An interesting alternative is to create a pup plugin for [asdf-vm](https://asdf-vm.com).

**Comment by dubrowgn** [comment]
As @merwok points out, this is already in the upstream debian repos, but it would be super handy to have it in the Ubuntu repos as well (and by extension derivative distros like Linux Mint). Any idea what that would take?

**Comment by jxu** [comment]
Someone to submit it to ubuntu repos and be a package maintainer
